model.py

```diff
@@ -0,0 +1,82 @@
+from matplotlib.pylab import f
+import numpy as np
+import pandas as pd
+import matplotlib as plib
+import torch
+from torch import nn
+
+class Generator(nn.Module):
+    def __init__(self, color_dim, out_dim, device):
```

Comment:
color_dim nije bas opis ove promenljive vise da je mozda nazoves da je to dimenzija filtera ili broj kanala suma/embedinga kasnije

Reply:
reseno